types.go

```diff
@@ -20,13 +20,14 @@ type Index struct {
 
 // Settings is the type that represent the settings in MeiliSearch
 type Settings struct {
-	RankingRules         []string            `json:"rankingRules,omitempty"`
-	DistinctAttribute    *string             `json:"distinctAttribute,omitempty"`
-	SearchableAttributes []string            `json:"searchableAttributes,omitempty"`
-	DisplayedAttributes  []string            `json:"displayedAttributes,omitempty"`
-	StopWords            []string            `json:"stopWords,omitempty"`
-	Synonyms             map[string][]string `json:"synonyms,omitempty"`
-	AcceptNewFields      bool                `json:"acceptNewFields,omitempty"`
+	RankingRules          []string            `json:"rankingRules,omitempty"`
```

Comment:
Does `omitempty` mean that it can be empty? 🙂

Reply:
In go every variable is automatically initialized to a default zero value (an int is 0, a string is "", a pointer will be nil, etc...). This means that when you create a struc, every field is initialized. For example:

```go
type struct Person {
    Size     int   `json:"size,omitempty"`
    Age      int   `json:"age,omitempty"`
}

```

and you create a Person object not defining on element (Age for example) like this:

```go
person := Preson { Size: 170 }
```

it will be initialized with the default zero values for each property (it would be the same as saying Person { Size: 170, Age 0 }

Reply:
Thansk!!!